code-studio/src/js/initApp/loadApp.js

```diff
@@ -106,6 +106,10 @@ module.exports = function (callback) {
         }
       }
 
+      if (progress.refreshStageProgress) {
```

Comment:
Just adding caution since it relies on header code having run previously which causes the refreshStageProgress to get created at runtime